packages/getting-started/src/browser/getting-started-widget.tsx

```diff
@@ -111,6 +111,14 @@ export class GettingStartedWidget extends ReactWidget {
         this.update();
     }
 
+    protected override onActivateRequest(msg: Message): void {
+        super.onActivateRequest(msg);
+        const elArr = document.getElementsByTagName('a');
```

Comment:
You shouldn't use `document.getElementsByTagName` because you can't be sure that some other widget won't contain an `a` tag.

```suggestion
        const elArr = this.node.getElementsByTagName('a');
```